pylint/checkers/strings.py

```diff
@@ -499,8 +497,13 @@ def _check_new_format_specifiers(self, node, fields, named):
                                          node=node)
                         break
 
+                inferred = previous.infer()
+                if inferred is astroid.Uninferable:
```

Comment:
pylint/test/functional/string_formatting_failed_inference.py Errors out with `TypeError: 'Uninferable' object is not an iterator`